tokens/delivery/http/tokens_delivery.go

```diff
@@ -123,11 +123,15 @@ func (a *TokensHandler) GetMetadata(c echo.Context) (err error) {
 }
 
 // @Summary Get prices
-// @Description Given a list of base denominations, returns the spot price with a system-configured quote denomination.
+// @Description Given a list of base denominations, this endpoint returns the spot price with a system-configured quote denomination.
+// If the pricing source is set to "chain" (0), it will first check the **chain** pricing cache for the price quote. If it exists, it will return it. Otherwise, it will compute the pricing using block data received from the Osmosis node and store it in the cache with an expiration time specified in the config.json file.
+// If the pricing source is set to "coingecko" (1), a similar pattern applies. It will look for the price quote in the **coingecko** pricing cache. If it exists, it will return it. Otherwise, it will fetch the price from the Coingecko API endpoint and store it in the cache with an expiration time specified in the config.json file.
+// If the token price is not available in the chain pricing source, it will fallback to the Coingecko pricing source if the quote denomination (human or chain) is USDC.
```

Comment:
This is not quite accurate for chain pricing.

With chain pricing and system-configured quote denomination, we pre-compute prices in the background by detecting tokens that were updated within a block.

As a result, we never expire their TTL.

Since we do not expose the ability for specifying quote denoms, this is the only possible behavior today

Reply:
https://github.com/osmosis-labs/sqs/blob/de34d172f95b221217967799f233c52181cfa07e/README.md#pricing

Added some documentation on this ^

In this PR: https://github.com/osmosis-labs/sqs/pull/214